hrs-portlets-webapp/src/main/webapp/WEB-INF/jsp/benefitInformation.jsp

```diff
@@ -24,27 +24,36 @@
 
 <div id="${n}dl-benefit-summary" class="fl-widget portlet dl-benefit-summary hrs">
   <div class="dl-banner-links">
-    <c:choose>
-    <c:when test="${enrollmentFlag == 'O'}">
+    <sec:authorize ifAnyGranted="ROLE_VIEW_NEW_HIRE_BENEFITS">
       <div class="dl-banner-link">
-        <spring:message code="benefit.summary.enrollment.O.message" text="You have a benefit enrollment opportunity. Please enroll online by clicking the following link: "/>
-        <a target="_blank" href="${hrsUrls['Open Enrollment/Hire Event']}"><spring:message code="open.enrollment" text="Open Enrollment" /></a>
-        <c:if test="${isMadisonUser}">
-          ${learnMoreEBenefitGuide}
-        </c:if>
+        You have a benefit enrollment opportunity.
+        <a target="_blank" href="${hrsUrls['Open Enrollment/Hire Event']}">Enroll now</a>
+        <c:choose>
+          <c:when test="${isMadisonUser}">
+            <a target="_blank" href="https://hr.wisc.edu/benefits/new-employee-benefits-enrollment/">Learn more</a>
+          </c:when>
+          <c:otherwise>
+            <a target="_blank" href="https://www.wisconsin.edu/ohrwd/benefits/">Learn more</a> 
+          </c:otherwise>>
+        </c:choose>
       </div>
-    </c:when>
-    <c:when test="${enrollmentFlag == 'H'}">
+    </sec:authorize>
+
+    <sec:authorize ifAnyGranted="ROLE_VIEW_OPEN_ENROLL_BENEFITS">
```

Comment:
I *suspect* this will work just fine, because

I *suspect* this role will be suppressed and the employee will only have the  `ROLE_VIEW_NEW_HIRE_BENEFITS` in the case where per policy the employee is entitled to both an event-driven and a calendar-driven enrollment opportunity. I suspect this because that's the way `enrollmentFlag` worked historically -- it's a one-character field and the event-driven `H` took precedence over the annual `O`. *However*, if it turns out employees can have both roles, then this rule might need to be more complicated to avoid duplicate "You have a benefit enrollment opportunity." messaging that I suppose would be technically correct but would also be confusing to actual employees. 

So, a nuance to keep an eye on as this goes to integration testing / prod. I wouldn't hold up merging this PR waiting on this clarification, easy enough to fix later if it turns out it needs fixing.